.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
-dist/
```

Comment:
why remove this? the dist should never be pushed

Reply:
I need to install the lib directly from GitHub. so without dist/index.js will fail to run. but I agree with you, had better do not push dist into git repo. so I remove the dist and workaround my problem by creating a private branch for myself. if you have any better solution will appreciate :)

Reply:
Hello, guys! Sorry for perhaps a stupid question, but why do you use this `distr` folder?
I am using your library, but had to fork it and change this main: `distr` to `src` in order get it work. What's reson for that?
I suspect that I need to run some build command seprartely for this module?

Reply:
You probably should open a new issue with the log you're getting. You shouldn't have to fork it 😄